test/e2e/storage/drivers/in_tree.go

```diff
@@ -1587,8 +1601,9 @@ func (a *awsDriver) GetDynamicProvisionStorageClass(config *testsuites.PerTestCo
 	}
 	ns := config.Framework.Namespace.Name
 	suffix := fmt.Sprintf("%s-sc", a.driverInfo.Name)
+	delayedBinding := storagev1.VolumeBindingWaitForFirstConsumer
 
-	return testsuites.GetStorageClass(provisioner, parameters, nil, ns, suffix)
+	return testsuites.GetStorageClass(provisioner, parameters, &delayedBinding, ns, suffix)
```

Comment:
do we still want to "turn on delayed binding" for these suites in this PR?

Reply:
No, we don't. Removed

Reply:
still here :)